index.html

```diff
@@ -34,27 +34,380 @@
+                <div class="title">
+                    <div class="imagelable"><img src="./bullets/Rectangle 55.png" alt=""></div>
+                    <div><h1>Language</h1></div>
+
+                </div>
+                <div class="below">
+                    <ul>
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                    </ul>
+                </div>
+                <div class="title">
+                    <div class="imagelable"><img src="./bullets/Rectangle 56.png" alt=""></div>
+                    <div><h1>Frameworks</h1></div>
+
+                </div>
+                <div class="below">
+                    <ul>
+                        <li>React.js</li>
+                        <li>Ruby on rails</li>
+                        <li>RSpec</li>
+                        <li>Capybara</li>
+                    </ul>
+                </div>
+                <div class="title">
+                    <div class="imagelable"><img src="./bullets/Rectangle 57.png" alt=""></div>
+                    <div><h1>Skills</h1></div>
+
+                </div>
+                <div class="below">
+                    <ul>
+                        <li>Database Management</li>
+                        <li>Version Control</li>
+                        <li>CLI</li>
+                        <li>API Design</li>
+                        <li>Web Development</li>
+                    </ul>
+                </div>
+
+
+              </div>
+
+
+            </main>
+      <!-- <footer class="bottom">
+         <div class="bottomwords">
+            <p>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+         </div>
+          <div class="form">
+            <form action="">
+                <label for="name">Full name</label> <br>
+                <input type="text" name="name">
+                <br>
+                <label for="email">Email address</label> <br>
+                <input type="email" name="email">
+                <textarea name="" id="" cols="30" rows="10" placeholder="write me something...."></textarea>
+                
+            </form>
+          </div>
+          
+  
+         <div class="footerbutton">
+            <button><a href="">Get in touch</a></button>
+         </div>
+  
+          <hr>
+          <ul class="footericons">
+              <li>
+                <a href="#"><img src="./bullets/git.png" alt="" /></a>
+              </li>
+              <li>
+                <a href="#"><img src="./bullets/Icon - Linkedin.png" alt="" /></a>
+              </li>
+              <li>
+                <a href="#"><img src="./bullets/Vector.png" alt="" /></a>
+              </li>
+              <li>
+                <a href="#"><img src="./bullets/twit.png" alt="" /></a>
+              </li>
+              <li>
```

Comment:
- [ ] Please remove all comment pieces of code in all your documents, this will keep your code clean and professional.